pkg/admission/pluginpreset_webhook_test.go

```diff
@@ -58,7 +58,8 @@ var _ = Describe("PluginPreset Admission Tests", Ordered, func() {
 				Namespace: test.TestNamespace,
 			},
 			Spec: greenhousev1alpha1.PluginPresetSpec{
-				ClusterSelector: metav1.LabelSelector{MatchLabels: map[string]string{"foo": "bar"}},
+				ClusterSelector:        metav1.LabelSelector{MatchLabels: map[string]string{"foo": "bar"}},
+				ClusterOptionOverrides: []greenhousev1alpha1.ClusterOptionOverride{},
```

Comment:
This won't be necessary after making the field optional. The overrides should not be required